lib/player.js

```diff
@@ -990,6 +1057,22 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return this.wrapWalkerListenersWithPromise_(events);
   }
 
+
+  /**
+   * Tell the player to use <code>canvas</code> for all <code>load</code>
+   * requests until <code>detach</code> or <code>destroy</code> are called.
```

Comment:
This comment appears to have some copy-paste bug from attach(), since the canvas isn't used directly by load(), and it isn't affected by detach().

Reply:
Removed this